### PR TITLE
made tests from es6-features-practise

### PR DESCRIPTION
**Pull request by OlgaNest95**
Aliaksei, please check.

### CODE REVIEW

File: es6-features-practise/test/iterators.js [hunk 1]
```diff
@@ -122,23 +132,23 @@ describe('A simple iterable without items inside, implementing the right protoco
...
-      let values;
+      let values='';
       for (let value of iterable) {
         values += value;
       }
       assert.equal(values, '');
     });
     it('has no `.length` property', function() {
-      const hasLengthProperty = iterable;
+      const hasLengthProperty =  'length' in iterable ;
```
Comment (OlgaNest95) [inline es6-features-practise/test/iterators.js:142]: I dont understand this one.

File: es6-features-practise/test/templateStrings.js [hunk 2]
```diff
@@ -17,25 +17,25 @@ describe('A template string, is wrapped in ` (backticks) instead of \' or "', fu
...
-      var evaluated = `${ x } + ${ y }`;
+      var evaluated = `${x+y}`;
       assert.equal(evaluated, x+y);
     });
     it('inside "${...}" can also be a function call', function() {
       function getDomain(){ 
         return document.domain; 
       }
-      var evaluated = `${ getDomain }`;
+      var evaluated = `${getDomain()}`;
```
Comment (OlgaNest95) [inline es6-features-practise/test/templateStrings.js:38]: Test check did not accept this task.

File: es6-features-practise/test/iterators.js (hunk 1 above)
Comment (tr3v3r) [reply es6-features-practise/test/iterators.js:142]: You need to check whether length property exists.

you can `iterable.hasOwnProperty('length')
or simply `typeof iterable.length === 'number' `

File: es6-features-practise/test/symbol.js [hunk 3]
```diff
@@ -9,26 +9,26 @@ var assert = require('assert');
...
 describe('Symbol', function() {
   it('`Symbol` lives in the global scope', function(){
-    const expected = someNamespace.Symbol;
+    const expected = Symbol;
     assert.equal(Symbol, expected);
   });
   it('every `Symbol()` is unique', function(){
     const sym1 = Symbol();
-    const sym2 = sym1;
+    const sym2 = 'symbol';
```
Comment (tr3v3r) [inline es6-features-practise/test/symbol.js:17]: `const sym2 = Symbol()`

File: es6-features-practise/test/symbol.js [hunk 4]
```diff
@@ -41,32 +41,32 @@ describe('Symbol', function() {
...
   it('stores the symbol in a runtime-wide registry and retrieves it from there', function() {
     const sym = Symbol.for('new symbol');
-    const sym1 = Symbol.for('new symbol1');
+    const sym1 = Symbol.for('new symbol');
     assert.equal(sym, sym1);
   });
   it('is different to `Symbol()` which creates a symbol every time and does not store it', function() {
-    var globalSymbol = Symbol.for('new symbol');
-    var localSymbol = Symbol.for('new symbol');
+    var globalSymbol = Symbol('new symbol');
```
Comment (tr3v3r) [inline es6-features-practise/test/symbol.js:53]:
```
var globalSymbol = Symbol.for('new symbol');
var localSymbol = Symbol('new symbol');
```

File: es6-features-practise/test/symbol.js [hunk 5]
```diff
@@ -81,22 +81,22 @@ describe('`Symbol.for` for registering Symbols globally', function() {
...
   });
   it('predefined symbols are not in the runtime-wide registry either', function() {
-    const key = Symbol.keyFor(Symbol.iteraTor);
+    const key = Symbol.keyFor(Symbol.iterator);
     assert.equal(key, void 0);
   });
   it('for non-Symbols throws an error', function() {
     function fn() {
-      Symbol.keyFor(Symbol.for('foo'));
+       new Symbol.keyFor(Symbol.for('foo'));
```
Comment (tr3v3r) [inline es6-features-practise/test/symbol.js:99]: Symbol.keyFor('foo');

File: es6-features-practise/test/templateStrings.js [hunk 6]
```diff
@@ -17,25 +17,25 @@ describe('A template string, is wrapped in ` (backticks) instead of \' or "', fu
...
 
   describe('can evaluate any expression, wrapped inside "${...}"', function() {
     it('all inside "${...}" gets evaluated', function() {
-      var evaluated = `${ x } + ${ y }`;
+      var evaluated = `${x+y}`;
       assert.equal(evaluated, x+y);
     });
     it('inside "${...}" can also be a function call', function() {
       function getDomain(){ 
         return document.domain;
```
Comment (tr3v3r) [inline es6-features-practise/test/templateStrings.js:36]: return 'tddbin.com'

**Review by tr3v3r — APPROVED** [review]

**Pull request merged**